pennylane/workflow/jacobian_products.py

```diff
@@ -401,8 +401,11 @@ def __init__(
         execution_config: Optional["qml.devices.ExecutionConfig"] = None,
```

Comment:
```suggestion
        execution_config: qml.devices.ExecutionConfig = qml.devices.DefaultExecutionConfig,
```

Reply:
That wouldn't work because of circular imports 😅

Reply:
I already switch the None to the default config in the body of the function